packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -50,22 +51,22 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
         // See http://crbug.com/715363
         // We use this flag to work-around this issue.
         '--disable-dev-shm-usage',
+        '--source-map=false',
+        '--force-gpu-mem-discardable-limit-mb',
+        //  set memory allocated for browse https://github.com/karma-runner/karma-chrome-launcher/issues/154#issuecomment-569698577
+        //  no super sure this is current anymore
+        '--max-old-space-size=8192',
```

Comment:
hmph what's the difference between this and what they have in their comment:
```
"--js-flags=--max-old-space-size=<size>"
```
maybe we should have the `--js-flags` arg prefixing it, cus `--max-old-space-size`.

after looking at chrome flags, it doesn't seem to be an argument listed [here](https://peter.sh/experiments/chromium-command-line-switches/) However it is found in the possible js-flags [here](https://chromium.googlesource.com/v8/v8/+/refs/heads/main/src/flags/flag-definitions.h#188) just gotta `ctrl + f` the term `max_old` to find it

Reply:
also curious about how this flag affects things:
```
"--remote-debugging-port=9222", // Without a remote debugging port, Google Chrome exits immediately.
```

Reply:
as far as max space vs old space, here's an [article](https://deepu.tech/memory-management-in-v8/) I just read a little bit of, mentions the `max-old-space-size` flag. makes sense since everything moves there that's survived 2 GC cycles, which I imagine is a lot of stuff by the later tests.

Also Addy Osmani has a great video on it memory in general. Starts going into the difference between new and old memory here: https://youtu.be/LaxbdIyBkL0?t=2341

Reply:
> also curious about how this flag affects things:

I think `karma-chrome-launcher` adds this by default if it's missing.

Reply:
> hmph what's the difference between this and what they have in their comment:

CMIIW but `--max-old-space-size` is a V8 (node) flag, and something like `--disable-dev-shm-usage` is a Chromium flag, that's why you need to prefix the first one with `--js-flags=` so it will be passed down to V8.